pkg/handlers/generic/mutation/extraapiservercertsans/inject.go

```diff
@@ -63,24 +66,33 @@ func (h *extraAPIServerCertSANsPatchHandler) Mutate(
 		h.variableFieldPath...,
 	)
 	if err != nil {
-		return err
+		log.Error(
```

Comment:
we should revert this since we are not reading cluster config variable anymore
```suggestion
		return err
```